channels/mx.m3u

```diff
@@ -115,8 +105,6 @@ http://wms.tecnoxia.com:1935/rytqrolive/rytqrolive/master.m3u8?GrupoTopIptv
 http://streamingcws20.com:1935/sqcs/videosqcs/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/FHiX8SL.png" group-title="",Tele Yucatan
 http://streamingcws20.com:1935/trecevision/videotrecevision/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/whNDApb.png" group-title="",Telebachillerato
-https://srvstream01.televisioneducativa.gob.mx/hls/telebachillerato.m3u8
```

Comment:
Link works (Tested in VLC)

Reply:
Still not working. VLC output:
<img width="740" alt="Screenshot 2019-10-12 at 20 18 06" src="https://user-images.githubusercontent.com/7253922/66705307-73958200-ed2d-11e9-9399-6a144ea51355.png">

Reply:
Still works for me